Core/include/Acts/Seeding/SeedFinder.hpp

```diff
@@ -63,9 +64,15 @@ class SeedFinder {
         candidates_collector;
 
     // managing doublet candidates
-    boost::container::small_vector<Acts::Neighbour<external_spacepoint_t>, 9>
+    boost::container::small_vector<
+        Acts::Neighbour<Acts::CylindricalSpacePointGrid<external_spacepoint_t>>,
```

Comment:
Just for legibility it might be worth typedefing the grid type above this line and reusing it in the `topNeighbours`.

Reply:
donve